google/cloud/gkehub/quickstart/README.md

```diff
@@ -157,7 +157,7 @@ set GRPC_DEFAULT_SSL_ROOTS_FILE_PATH=%cd%\roots.pem
 ```
 
 [bazel-install]: https://docs.bazel.build/versions/main/install.html
-[quickstart-link]: https://cloud.google.com/gkehub/docs/quickstart
+[quickstart-link]: https://cloud.google.com/kubernetes-engine/docs/quickstart
```

Comment:
I would use: https://cloud.google.com/anthos/multicluster-management/connect/how-to